pkg/flock/flock_int_test.go

```diff
@@ -0,0 +1,33 @@
+package flock_test
+
+import (
+	"testing"
+
+	. "github.com/onsi/ginkgo"
+	. "github.com/onsi/gomega"
+	"github.com/rancher/k3s/pkg/flock"
+)
+
+var lockfile string = "/tmp/testlock.test"
```

Comment:
Is this value ever updated? If not, can you make it a `const`?

Reply:
Made const